docs/classic-ui/templates-global-variables.md

```diff
@@ -0,0 +1,21 @@
+---
+html_meta:
+  "description": "Convenience global Plone variables in templates"
+  "property=og:description": "Convenience global Plone variables in templates"
+  "property=og:title": "Global variables in templates"
+  "keywords": "Plone, Classic UI, global variables, templates"
+---
+
+# Global variables in templates
+
+For convenience Plone defines a couple of global variables often used in templates.
+
+- [`portal_state`](backend-global-utils-portal-state-label)
+- [`context_state`](backend-global-utils-context-state-label)
+- [`plone_view`](backend-global-utils-plone-view-label)
```

Comment:
I missed this on my first pass. The above three items should use MyST syntax:

```
- {ref}`backend-global-utils-portal-state-label`
- {ref}`backend-global-utils-context-state-label`
- {ref}`backend-global-utils-plone-view-label`
```

Reply:
why? it was working, i did i miss something? What is the difference?

Reply:
Both work. The MyST syntax is cleaner. I'm actually surprised that the `[foo](bar)` (Markdown) syntax even worked, so I am mildly concerned that it might not work in the future. I am not sure if that syntax is a deliberate or unintended feature.

Reply:
I would argue in the oposite direction here, the markdown sytax is well known and is explicite whioch the special myst is not.
The advantage of having support of both RST and MD is that people can just use what they know, i think as long as it works in our stack we should allow that.
what say you?

Reply:
also the MyST syntax forces me to have excactly the referenced title, in MD i can use the reference in a sentence a have a slightly different wording which fit's better into the sentence.

Reply:
MyST syntax is not special. Its syntax is derived from reST.

Both MyST and reST have an advantage of not requiring the insertion of link text. The following examples will render the section heading hyperlinked to the target. If the heading ever changes, then the rendering will automatically update, too. This is one of reST's and MyST's super powers.

```
# reST
:ref:`backend-global-utils-portal-state-label`
# MyST
{ref}`backend-global-utils-portal-state-label`
```

If you want custom link text for the label, you can do that in MyST and reST, too.

```
# reST
:ref:`link text <target-label>`
# MyST
{ref}`link text <target-label>`
```

You cannot format link text in reST, but you can in Markdown.

You cannot automatically render the text of the target in Markdown. This will render anchor markup, but without any link text.

`[](backend-global-utils-portal-state-label)`

You either must know what is the correct name of the target, or you can use custom text. For the former, if that section heading name changes, you have to change the text in the link as well.

You are not forced to do one thing or another. You have the choice to use the tool that best does the job. That's the beauty of MyST: it combines features from both reST and Markdown. You can use whatever works.

When I want to automatically keep the rendered headings in synch with their references, I use MyST or reST. When I want easier syntax for custom link text, or formatting of link text, I use Markdown.

In this specific instance, because it is a list of section headings, and not in a sentence, I would go with reST-like MyST syntax for ease of future maintenance.

Reply:
thx for the clearifcation, I'll do so.